wallet/src/de/schildbach/wallet/ui/EnableFingerprintDialog.java

```diff
@@ -1,156 +0,0 @@
-/*
- * Copyright 2018 Dash Core Group
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package de.schildbach.wallet.ui;
-
-import android.app.Activity;
-import android.app.Dialog;
-import android.content.DialogInterface;
-import android.os.Bundle;
-import android.view.LayoutInflater;
-import android.view.View;
-
-import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
-import androidx.core.os.CancellationSignal;
-import androidx.fragment.app.FragmentManager;
-import androidx.lifecycle.ViewModelProvider;
-
-import org.dash.wallet.common.ui.BaseDialogFragment;
-
-import de.schildbach.wallet.WalletApplication;
-import de.schildbach.wallet.ui.widget.FingerprintView;
-import de.schildbach.wallet.util.FingerprintHelper;
-import de.schildbach.wallet_test.R;
-import kotlin.Pair;
-import kotlin.Unit;
-
-/**
- * @author Samuel Barbosa
- */
-public class EnableFingerprintDialog extends BaseDialogFragment {
```

Comment:
This dialog isn't needed anymore since `biometric` has it's own that often overlaps this one.
We can use a simple `AdaptiveDialog` to ask users if they want to enable the fingerprint scanner.